examples/evm/call-contract/CallContract.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { AxelarExecutable } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/executable/AxelarExecutable.sol';
+import { IAxelarGateway } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol';
+import { IAxelarGasService } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGasService.sol';
+import { IERC20 } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol';
+
+
+/**
+ * @title CallContract
+ * @author Axelar
+ * @notice Send a message from chain A to chain B and stores gmp message
+ */
+contract CallContract is AxelarExecutable {
```

Comment:
Do we feel like CallContract is the right name, given that this initiates and receives a call? I'm OK with it, just want to make sure we're thinking through it.

Reply:
Ya, I thought about this I think its most intuitive. I.e. it follows the name of the folder that it's in, and when deciding about a name I just thought about what we typically refer to this as, which is the CallContract example (not ExecutableSample as it was named before)